docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md

```diff
@@ -7,7 +7,7 @@ weight = 400
 
 # Create a Cortex or Loki managed recording rule
 
-You can create and manage recording rules for an external Cortex or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh. 
+You can create and manage recording rules for an external Cortex or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
```

Comment:
👁️‍🗨️ Note: Because I had to fix conflicts with Drone between my branch and the main branch, the linter for newer `md` files was triggered.